core/src/org/samtools/htsjdk/core/Locatable.java

```diff
@@ -0,0 +1,140 @@
+package org.samtools.htsjdk.core;

```

Comment:
How do you feel by making a package that is called `org.samtools.htsjdk.core.api` for the core interfaces and then a implementation-specific package? For example (just care about the api name, the rest are just first idea of how to name specific implementations):

* `Read` interface in the `api` package, and `SAMRecord` implementation in `org.samtools.htsjdk.core.sam`
* `Variant` interface in the `api` package, and `VCFRecord` implementation in `org.samtools.htsjdk.core.vcf`